IzvorniKod/Services/CalendarService.cs

```diff
@@ -25,6 +25,13 @@ public async Task UploadCalendarAsync(IFormFile file, string userId)
     {
         if (file != null && file.Length > 0)
         {
+            
+            var fileExtension = Path.GetExtension(file.FileName);
+            if (fileExtension != ".csv")
+            {
+                throw new InvalidOperationException("Invalid file type. Only .csv files are allowed.");
```

Comment:
Napraviti cu item za ovo ali handleanje ovih exeptiona bi se trebalo srediti u nekom trenutku, u redu je pustiti bez toga

Reply:
Zamisao je imati middleware koji handle-a exceptione, ako je moguće. To be decided.

Reply:
okej zvuci good